src/components/dialogs/network-modifications/two-windings-transformer/creation/characteristics-pane/two-windings-transformer-creation-characteristics-pane.js

```diff
@@ -81,34 +78,20 @@ const TwoWindingsTransformerCreationCharacteristicsPane = ({
                 }
                 p={1}
             >
-                <Grid container spacing={2}>
-                    <Grid item xs={12}>
-                        <h3 className={classes.h3}>
-                            <FormattedMessage id="Connectivity" />
-                        </h3>
-                    </Grid>
-                </Grid>
-                <Grid container spacing={2}>
-                    <Grid item xs={6}>
-                        <h4 className={classes.h4}>
-                            <FormattedMessage id="Side1" />
-                        </h4>
-                    </Grid>
-                </Grid>
+                <GridSection
+                    title="Connectivity"
+                    heading="3"
```

Comment:
it's already the default value, no need to specify it

Reply:
Thanks. Fixed.